src/DurableTask.Netherite/OrchestrationService/NetheriteOrchestrationService.cs

```diff
@@ -290,6 +290,10 @@ async Task IOrchestrationService.DeleteAsync()
         /// <inheritdoc />
         Task IOrchestrationService.StartAsync()
         {
+            // We discard exception state, as Functions Host may retry the StartAsync operation.
```

Comment:
I suggest mentioning the Functions Host as an example of a host that retries StartAsync failures, since the problem we're fixing is not necessarily specific to Azure Functions (nor is this Netherite code).

Reply:
Good point. I've called this out in the comments.

Reply:
resolving as this PR was "taken over".